code/api/src/seeders/modern.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const params = require('../config/params');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('surveys', [
+      // Mens
+      {
+        type: 'bottom',
+        image: 'code/api/public/images/survey/modern/mens_pants_modern.jpg',
+        score: [5,0],
+        createdAt: new Date(),
```

Comment:
Nice, it is on the correct axis, which took an embarrassingly long time for me to remember which was x and which was y.